src/pages/blog/remembering-dan-kohn.md

```diff
@@ -3,7 +3,7 @@ templateKey: blog-post
 title: >-
   Remembering Dan Kohn
 author: Jonathan Bryce
-date: 2020-11-6T13:30:56.289Z
+date: 2020-19-6T13:30:56.289Z
```

Comment:
The date format here is YYYY-MM-DDTHH:MM:SS.MMMZ. I would probably not edit the date since it's when we posted it and you mention in the body of the post when the update is made